Backend/src/API/REST/routes/institution/institutionRouter.ts

```diff
@@ -307,7 +307,7 @@ const disjoin_member_of_institution = (): Router => {
         return res.status(409).send("Information of the user does not match.");
 
       const institution = await InstitutionService.getInstitutionById(institutionId);
-      if (!institution) return res.status(409).send("Institution does not exist.");
+      if (!institution) return res.status(400).send("Institution does not exist.");
```

Comment:
I want to mention that 400 error code means that it is a “Bad Request.” The server can’t return a response due to an error on the client’s end.
Whereas 404: means “The requested resource was not found.” This is the most common error message of them all. This code means that the [requested resource does not exist] and the server does not know if it ever existed.

Since you addressed the error code change here, in this PR maybe you can change it to 404.